spec/services/notify_app_store/message_builder_spec.rb

```diff
@@ -121,7 +121,17 @@
             'uplift' => nil,
             'work_type' => { en: an_instance_of(String), value: work_item.work_type },
           }],
-          'youth_count' => 'no'
+          'youth_count' => 'no',
```

Comment:
I thought this is **Youth Court**

Reply:
Added to https://dsdmoj.atlassian.net/browse/CRM457-645